packages/blog/db/db_ContentMigration.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE blogposts ADD COLUMN if not exists blogpostcontent jsonb;
```

Comment:
Could you please move this file (and the other db sql files) to idealo.design directory (e.g. `packages/idealo.design/src/server/db`). I was under the impression that we wanted to delete the blog directory soon.

Reply:
Thank you @nforgerit for the review, we will move the db-sql-files to 'idealo.design'-directory so the blog directory can be eventually deleted!